tests/featurecounts_group_type.nf.test

```diff
@@ -1,15 +1,15 @@
 nextflow_pipeline {
 
-    name "Test pipeline with featureCounts with group type false"
+    name "Test pipeline with --featurecounts_group_type false"
     script "../main.nf"
-    tag "rnaseq"
+    tag "RNASEQ"
     tag "PIPELINE"
 
-    test("--featurecounts_group_type false") {
+    test("Parameters: --featurecounts_group_type false") {
```

Comment:
If you run the test without the `Parameters:` it isn't entirely obvious as to what test you are running in the terminal:

![image](https://github.com/nf-core/rnaseq/assets/23529759/abbded6a-3fa0-42c4-90fe-838dec8ce638)

There is this `--skip_qc` option that looks like a parameter, but I think it would be nicer to make it more obvious. I truncated `Parameters` to `Params`. See https://github.com/nf-core/rnaseq/pull/1324/commits/100ae0a839fdd300f31d0a12b20e835dc67b0861